scale-tests/configs/2018-08-22-mvdrf112s.env

```diff
@@ -0,0 +1,127 @@
+NUM_DISPATCHERS="$((${NON_GPU_NUM_DISPATCHERS} + ${GPU_NUM_DISPATCHERS}))"
+DISPATCHERS_JSON_OUTPUT_FILE="${TEST_NAME}-all-dispatchers-${NUM_DISPATCHERS}.json"
+
+# Failing streaming jobs configuration #########################################
+
+FAILING_SUBMISSIONS_OUTPUT_FILE="${TEST_NAME}-failing-submissions.out"
+FAILING_NUM_PRODUCERS_PER_KAFKA="${NUM_DISPATCHERS}" # 1 Kafka and 50 dispatchers -> 50 producers.
+FAILING_NUM_CONSUMERS_PER_PRODUCER=1 # 50 producers -> 50 consumers.
+FAILING_PRODUCER_NUMBER_OF_WORDS=7692
+FAILING_PRODUCER_WORDS_PER_SECOND=1
+FAILING_PRODUCER_SPARK_CORES_MAX=2
+FAILING_PRODUCER_SPARK_EXECUTOR_CORES=2
+FAILING_CONSUMER_BATCH_SIZE_SECONDS=10
+FAILING_CONSUMER_SPARK_CORES_MAX=1
+FAILING_CONSUMER_SPARK_EXECUTOR_CORES=1
+
+# Finite streaming jobs configuration ##########################################
+
+FINITE_SUBMISSIONS_OUTPUT_FILE="${TEST_NAME}-finite-submissions.out"
+FINITE_NUM_PRODUCERS_PER_KAFKA="${NUM_DISPATCHERS}" # 1 Kafka and 50 dispatchers -> 50 producers.
+FINITE_NUM_CONSUMERS_PER_PRODUCER=1 # 50 producers -> 50 consumers.
+# 50 producers + 50 consumers = 100 total finite streaming jobs
+FINITE_PRODUCER_NUMBER_OF_WORDS=7692
+FINITE_PRODUCER_WORDS_PER_SECOND=1
+# 7692 words / 1 word per second -> ~2h runtime.
+FINITE_PRODUCER_SPARK_CORES_MAX=2
+FINITE_PRODUCER_SPARK_EXECUTOR_CORES=2
+FINITE_CONSUMER_BATCH_SIZE_SECONDS=10
+FINITE_CONSUMER_SPARK_CORES_MAX=1
+FINITE_CONSUMER_SPARK_EXECUTOR_CORES=1
+
+# Infinite streaming jobs configuration ########################################
+
+INFINITE_SUBMISSIONS_OUTPUT_FILE="${TEST_NAME}-infinite-submissions.out"
+INFINITE_NUM_PRODUCERS_PER_KAFKA="${NUM_DISPATCHERS}" # 1 Kafka and 50 dispatchers -> 50 producers.
+INFINITE_NUM_CONSUMERS_PER_PRODUCER=2 # 50 producers -> 100 consumers.
+# 50 producers + 100 consumers = 150 total infinite streaming jobs
+INFINITE_PRODUCER_NUMBER_OF_WORDS=0
+INFINITE_PRODUCER_WORDS_PER_SECOND=1
+INFINITE_PRODUCER_SPARK_CORES_MAX=2
+INFINITE_PRODUCER_SPARK_EXECUTOR_CORES=2
+INFINITE_CONSUMER_BATCH_SIZE_SECONDS=10
+INFINITE_CONSUMER_SPARK_CORES_MAX=1
+INFINITE_CONSUMER_SPARK_EXECUTOR_CORES=1
+
+# Batch jobs configuration #####################################################
+
+NON_GPU_DISPATCHERS_JSON_OUTPUT_FILE_URL="https://${TEST_S3_BUCKET}.s3.amazonaws.com/${TEST_S3_FOLDER}/${NON_GPU_DISPATCHERS_JSON_OUTPUT_FILE}"
+
+BATCH_APP_ID="/${SERVICE_NAMES_PREFIX}batch-workload"
+BATCH_SCRIPT_CPUS=6
+BATCH_SCRIPT_MEM=12288
+BATCH_SUBMITS_PER_MIN=13
+BATCH_SPARK_BUILD_BRANCH=master
+
+# Batch GPU jobs configuration #################################################
+
+GPU_DISPATCHERS_JSON_OUTPUT_FILE_URL="https://${TEST_S3_BUCKET}.s3.amazonaws.com/${TEST_S3_FOLDER}/${GPU_DISPATCHERS_JSON_OUTPUT_FILE}"
+
+GPU_APP_ID="/${SERVICE_NAMES_PREFIX}gpu-batch-workload"
+GPU_SCRIPT_CPUS=2
+GPU_SCRIPT_MEM=4096
+GPU_DOCKER_IMAGE='samvantran/spark-dcos-gpu:metrics'
+GPU_SUBMITS_PER_MIN=1
```

Comment:
can you change this to 5? This was accidentally set to 1 last time and resulted in a very low amount of gpu jobs. Setting to 5 should result in all gpu agents working majority of the time.

Reply:
Bumped to 5: https://github.com/mesosphere/spark-build/pull/395/commits/7fa53dccdb3d113cee431097876898d20700d537